scripts/lib/upgrade-zulip-stage-2

```diff
@@ -203,7 +203,7 @@ elif args.from_git:
     # update-prod-static with the git upgrade process.  But it'll fail
     # safely; this seems like a worthwhile tradeoff to minimize downtime.
     logging.info("Building static assets...")
-    subprocess.check_call(["./tools/update-prod-static", "--prev-deploy",
+    subprocess.check_call(["./tools/update-prod-static", "--prev-deploy=" +
                            os.path.join(DEPLOYMENTS_DIR, 'current')],
```

Comment:
Hmm, I think the old version was better for this one.

Reply:
Compare the post-Black versions:

```diff
--- a/scripts/lib/upgrade-zulip-stage-2
+++ b/scripts/lib/upgrade-zulip-stage-2
@@ -246,8 +246,7 @@
     subprocess.check_call(
         [
             "./tools/update-prod-static",
-            "--prev-deploy",
-            os.path.join(DEPLOYMENTS_DIR, "current"),
+            "--prev-deploy=" + os.path.join(DEPLOYMENTS_DIR, "current"),
         ],
         preexec_fn=su_to_zulip,
     )
```

The improvement is small in this case, but always using the `=` is consistent with other places where the improvement is larger, 

```diff
--- a/tools/webpack
+++ b/tools/webpack
@@ -36,14 +30,10 @@
     webpack_args += [
         # webpack-cli has a bug where it ignores --watch-poll with
         # multi-config, and we don't need the katex-cli part anyway.
-        "--config-name",
-        "frontend",
-        "--allowed-hosts",
-        ",".join([host, ".zulipdev.com", ".zulipdev.org"]),
-        "--host",
-        host,
-        "--port",
-        port,
+        "--config-name=frontend",
+        "--allowed-hosts=" + ",".join([host, ".zulipdev.com", ".zulipdev.org"]),
+        f"--host={host}",
+        f"--port={port}",
         # We add the hot flag using the cli because it takes care
         # of addition to entry points and adding the plugin
         # automatically
```

Reply:
Yeah, definitely agreed with the others; this one just felt a bit weird.  I'm OK with it I guess.